DigitalLearningSolutions.Data/DigitalLearningSolutions.Data.csproj

```diff
@@ -1,8 +1,9 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
     <TargetFramework>netcoreapp3.1</TargetFramework>
     <Nullable>enable</Nullable>
+    <UserSecretsId>cc50a716-0c33-41f2-aae9-55493a9f6437</UserSecretsId>

```

Comment:
Is the fact that this is a specific Guid going to be a problem? No idea how the secrets stuff works

Reply:
I don't think so, the docs say that this Guid is unique to the project rather than a user. But not totally sure
https://docs.microsoft.com/en-us/aspnet/core/security/app-secrets?view=aspnetcore-6.0&tabs=windows#secret-manager

Reply:
Yeah I think it'll be fine looking at the documentation. It creates a folder in appdata/.../UserSecrets with the Guid as the name, so we should be fine to all have the same folder name.

Reply:
We don't need this - the secrets are linked to the Web project, and won't be picked up as config using this key.